tests/functional/graph_selection/test_graph_selection.py

```diff
@@ -32,41 +32,23 @@ def clear_schema(project):
     project.run_sql("create schema {schema}")
 
 
-@pytest.fixture
-def run_seed(project):
-    clear_schema(project)
-    run_dbt(["seed"])
-
-
-@pytest.mark.usefixtures("project", "run_seed")
 class TestGraphSelection(SelectionFixtures):
+    # The tests here aiming to test whether the correct node is selected,
+    # we don't need the run to pass
     @pytest.fixture(scope="class")
     def selectors(self):
         return selectors_yml
 
     def test_specific_model(self, project):
-        run_dbt(["seed"])
-        results = run_dbt(["run", "--select", "users"])
-        assert len(results) == 1
-
-        check_relations_equal(project.adapter, ["seed", "users"])
-
-        created_tables = project.get_tables_in_schema()
-        assert "users_rollup" not in created_tables
-        assert "alternative.users" not in created_tables
-        assert "base_users" not in created_tables
-        assert "emails" not in created_tables
+        results = run_dbt(["run", "--select", "users"], expect_pass=False)
```

Comment:
Do we want this to just be `list`, rather than `run` + expect fail?

(feel free to ignore me and ship as is)

Reply:
That's a great idea! updating

Reply:
Actually `ls` and `run` doesn't seems to select the exact same items, it would include source, exposure etc. It would be much bigger change including change the actual expected results to make the update. I think we can leave it as an item if we plan to touch more things related to selector

Reply:
Oh good call. You can limit this to models (same as `run`) with `dbt ls --resource-type model`. But very happy with leaving it as is

Reply:
Ops, this poped up after I hit the merge button. This is good to know!